sdks/go/pkg/beam/core/util/hooks/hooks.go

```diff
@@ -0,0 +1,184 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Hooks allow runners to tailor execution of the worker to allow for customization
```

Comment:
<!--new_thread; commit:5d13d3df224a378bb27856f0351e23e0898e862b; resolved:0-->
Consider removing the blank line between this comment block and the package declaration, and changing this first sentence to:

Package hooks allows runners to tailor execution of the worker harness.

Reply:
Done.